ehrenamt-justiz-backend/README.md

```
@@ -177,4 +177,24 @@
+logging:
+  include-application-group: false # disabled because application group is only relevant when deployed
+  structured:
+    format:
+      console: # set to empty to disable structured logging locally
+  level:
+    root: INFO
+    de.muenchen.refarch: DEBUG
+    org.springframework.cache: TRACE
+    org.springframework.web: DEBUG
+    org.springframework.security: DEBUG
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)